packages/lib/server/i18n.ts

```diff
@@ -1,72 +1,49 @@
 import { createInstance } from "i18next";
 
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { i18n } = require("@calcom/config/next-i18next.config");
```

Comment:
@emrysal 
- If we use ESModule syntax here, packages/platform/libraries build gets broken.
- We already do this same pattern already apparently [here](https://github.com/calcom/cal.com/blob/main/packages/lib/getLocaleFromRequest.ts) and [here](https://github.com/calcom/cal.com/blob/main/packages/lib/i18n.ts)